bitnami/harbor/templates/core/core-dpl.yaml

```
@@ -0,0 +1,148 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "harbor.core" . }}
+  labels:
+    {{ include "harbor.labels" . | nindent 4 }}
+    app.kubernetes.io/component: core
+spec:
+  replicas: {{ .Values.core.replicas }}
+  selector:
+    matchLabels:
+      {{ include "harbor.matchLabels" . | nindent 6 }}
+      app.kubernetes.io/component: core
+  template:
+    metadata:
+      labels:
+        {{ include "harbor.matchLabels" . | nindent 8 }}
+        app.kubernetes.io/component: core
+      annotations:
+        checksum/configmap: {{ include (print $.Template.BasePath "/core/core-cm.yaml") . | sha256sum }}
+        checksum/configmap-envvars: {{ include (print $.Template.BasePath "/core/core-cm-envvars.yaml") . | sha256sum }}
+        checksum/secret: {{ include (print $.Template.BasePath "/core/core-secret.yaml") . | sha256sum }}
+        checksum/secret-envvars: {{ include (print $.Template.BasePath "/core/core-secret-envvars.yaml") . | sha256sum }}
+        checksum/secret-jobservice: {{ include (print $.Template.BasePath "/jobservice/jobservice-secrets.yaml") . | sha256sum }}
+        {{- if .Values.core.podAnnotations }}
+        {{ toYaml .Values.core.podAnnotations | nindent 8 }}
```

Use `{{-` to avoid adding unnecessary new lines
```suggestion
        {{- toYaml .Values.core.podAnnotations | nindent 8 }}
```
Apply this change to every template using this approach